src/lkl/syscall-overrides.c

```diff
@@ -1,6 +1,9 @@
+#include <sys/syscall.h>
 #include "lkl/syscall-overrides-futex.h"
 #include "lkl/posix-host.h"
+#include "enclave/enclave_mem.h"
 
 void register_lkl_syscall_overrides()
 {
+    lkl_replace_syscall(SYS_sysinfo, syscall_SYS_sysinfo);
```

Comment:
I think this is the wrong syscall number.  LKL has different syscall numbers to x86-64 Linux.  This needs to be `__lkl__NR_sysinfo`.  See the changes for the futex override in #162 for how to structure this.

Reply:
`SYS_sysinfo` has the same value as `__NR_sysinfo`. I couldn't find anywhere we define `__lkl__NR_sysinfo` though.